packages/app-storage/src/Query.tsx

```diff
@@ -41,11 +43,51 @@ class Query extends React.PureComponent<Props, State> {
         value as Storage$Key$Value
       );
 
-      cache[id] = withStorageDiv(key, { params: values })(
-        (value: any) =>
-          valueToText(key.type, value),
-        { className: 'ui--output' }
-      );
+      // FIXME - Move into ui-app/src/Params/valueToAccountList for reuse
+      const renderAccounts = (addresses: Array<Uint8Array | string>) => {
+        return (
+          <table className='accounts'>
+            <thead>
+              <tr>
+                <th>Identitycon</th>
```

Comment:
When displaying (since we should not be using tables, rather render the single values - see other comments), no need to have a header for Icon. (throw-away comment here sin ce the implementation needs some work)